main.py

```diff
@@ -156,12 +159,12 @@ def main():
                                 'price': last_price}
                             logger.info('Sold coins:\r\n' + sold_coins[coin])
 
-                            store_order('sold.json', sold_coins)
+                            store_order(settings.data_folder + '/sold.json', sold_coins)
 
         # the buy block and logic pass
-        # announcement_coin = load_order('new_listing.json')
-        if os.path.isfile('new_listing.json'):
-            announcement_coin = load_order('new_listing.json')
+        # announcement_coin = load_order(settings.data_folder + '/new_listing.json')
```

Comment:
Can this commented out block be removed altogether?

Reply:
yes

Reply:
yes